actions/views/rhs.test.js

```diff
@@ -228,128 +203,6 @@ describe('rhs view actions', () => {
-            );
-
-            compareStore.dispatch(batchActions(postRHSSearchActions));
-
-            expect(store.getActions()).toEqual(compareStore.getActions());
-        });
-    });
-
-    describe('getPinnedPosts', () => {
-        test('it dispatches the right actions', async () => {
-            await store.dispatch(getPinnedPosts());
-
-            const compareStore = mockStore(initialState);
-            const result = await Client4.getPinnedPosts(currentChannelId);
-            await PostActions.getProfilesAndStatusesForPosts(result.posts, compareStore.dispatch, compareStore.getState);
-
-            compareStore.dispatch(batchActions([
-                {
-                    type: SearchTypes.RECEIVED_SEARCH_POSTS,
-                    data: result,
-                },
-                {
-                    type: SearchTypes.RECEIVED_SEARCH_TERM,
-                    data: {
-                        teamId: '321',
-                        terms: null,
-                        isOrSearch: false,
-                    },
-                },
-                {
-                    type: SearchTypes.SEARCH_POSTS_SUCCESS,
-                },
-            ]));
-
-            expect(store.getActions()).toEqual(compareStore.getActions());
-        });
-    });
-
-    describe('showPinnedPosts', () => {
-        test('it dispatches the right actions', async () => {
-            store.dispatch(showPinnedPosts());
-
-            const compareStore = mockStore(initialState);
-            const result = await Client4.getPinnedPosts('123');
-            await PostActions.getProfilesAndStatusesForPosts(result.posts, compareStore.dispatch, compareStore.getState);
-
-            compareStore.dispatch(batchActions([
-                {
-                    type: ActionTypes.SEARCH_PINNED_POSTS_REQUEST,
-                },
-                {
-                    type: ActionTypes.UPDATE_RHS_SEARCH_TERMS,
-                    terms: '',
-                },
-                {
-                    type: ActionTypes.UPDATE_RHS_STATE,
-                    state: RHSStates.PIN,
-                    channelId: '123',
-                },
-            ]));
-
-            compareStore.dispatch(batchActions([
-                {
-                    type: SearchTypes.RECEIVED_SEARCH_POSTS,
-                    data: result,
-                },
-                {
-                    type: SearchTypes.RECEIVED_SEARCH_TERM,
-                    data: {
-                        teamId: '321',
-                        terms: null,
-                        isOrSearch: false,
-                    },
-                },
-                {
-                    type: SearchTypes.SEARCH_POSTS_SUCCESS,
-                },
-                {
-                    type: ActionTypes.SEARCH_PINNED_POSTS_SUCCESS,
-                },
-            ]));
-
-            expect(store.getActions()).toEqual(compareStore.getActions());
-        });
-    });
-
```

Comment:
Shouldn't we just modify tests for `showPinnedPosts` and `showFlaggedPosts` instead of deleting?

Reply:
I think I just deleted them because I don’t think they’re useful tests. They just feel like they’re just a copy-paste of the function body. You’re right that I probably shouldn’t just delete them, so I’ll take another shot at updating them